### DISCUSSION TITLE
Seeing Beyond Statistics: Anscombe's Quartet and the Power of Graphs

### DISCUSSION THREAD

**Discussion by jeroenjanssens** [has2k1/plotnine#818] [2024 Plotnine Contest]
### Authors

Jeroen Janssens

### Links

[blog](https://jeroenjanssens.com), [github](https://github.com/jeroenjanssens), [linkedin](https://www.linkedin.com/in/jeroenjanssens), [twitter](https://twitter.com/jeroenhjanssens) 

### Full description

<img align="right" width="320" height="240" src="https://raw.githubusercontent.com/jeroenjanssens/anscombes-quartet/main/anscombes-quartet.png">

In 1973, statistician Francis Anscombe challenged the commonly held belief that "numerical calculations are exact, but graphs are rough" (Anscombe, 1973). His strategy was to manually construct four small datasets---now known as Anscombe's Quartet---that have different distributions but nearly identical descriptive statistics. It's perhaps the best argument for visualizing data. 

In this notebook we replicate his paper by using Plotnine to visualize those datasets and Polars to verify their descriptive statistics. More specifically, we:

- Turn the original values that Anscombe used into one long DataFrame
- Verify the descriptive statistics
- Create a couple of exploratory data visualizations
- Fine-tune the visualization to make it suitable for communication

[Read more...](https://github.com/jeroenjanssens/anscombes-quartet/blob/main/README.md)

### Code repository

https://github.com/jeroenjanssens/anscombes-quartet

**Comment by has2k1** [comment]
<img width="461" alt="anscombes-quartet" src="https://github.com/user-attachments/assets/70d12b25-80a0-48ac-9ff6-04559fd53d27">

**Comment by has2k1** [comment]
@jeroenjanssens, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
